### DISCUSSION TITLE
Running maintenance failed of alertmanager

### DISCUSSION THREAD

**Discussion by adeel86** [grafana/grafana#63095] [Alerting]
I have downloaded the latest Grafana source code and set up my development environment. After running the frontend and backend, it starts showing up in the error logging. I haven't made any changes to the source code yet.

ERROR Running maintenance failed logger=alertmanager err="context canceled"

Can anyone explain what is causing this error?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
